docker/php/php-cli.dev.ini

```diff
@@ -0,0 +1,14 @@
+apc.enable_cli = 1
+date.timezone = ${PHP_DATE_TIMEZONE}
+opcache.enable_cli = 1
```

Comment:
Opcache in CLI might cause some issues during development

Reply:
Such as? :D

Reply:
Once we had an issue when phpspec was running unexisting spec which had been removed before, the issue vanished after we disabled opcache.

Reply:
To be honest, that sounds like a misconfiguration of OpCache, so we should be fine here.